### PR TITLE
Re-integrate old theme designer

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Copy the theme designer from the previous version of the website into a hidden page in the new website. 

Disclaimer: I did not write most of this code, just restored it from a deleted file.

It's a draft because it doesn't work yet. :P (it renders, then throws when you try to edit the theme)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8994)

### CODE REVIEW

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md [hunk 1]
```diff
@@ -9150,17 +9150,13 @@ export const TextView: ITextComponent['view'];
 
 // @public (undocumented)
 export class ThemeGenerator {
-    // (undocumented)
+    static ensureSlots(slotRules: IThemeRules, isInverted: boolean): void;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: 🤣

**Comment by dzearing** [comment]
no change file makes build busted. Please add.

**Comment by dzearing** [comment]
@atneik are you looking at the PRs? This is a week old without traction.

**Comment by ecraig12345** [comment]
@dzearing This is a draft currently because it's totally broken (not just due to the missing change file). I was hoping @jdhuntington could take a look at fixing it while I was on vacation, but since that didn't happen, I'll try and work on it this week.

**Pull request closed without merging**